BUILDING.md

```diff
@@ -131,3 +131,28 @@ When working with `ctr`, the containerd CLI we just built, don't forget to start
 ```sh
 containerd --config config.toml
 ```
+
+# Testing containerd
+
+During the automated CI the unit tests and integration tests are run as part of the PR validation. As a developer you can run these tests locally by using any of the following `Makefile` targets:
+ - `make test`: run all non-integration tests that do not require `root` privileges
+ - `make root-test`: run all non-integration tests which require `root`
+ - `make integration`: run all tests, including integration tests and those which require `root`
+ - `make integration-parallel`: run all tests (integration and root-required included) in parallel mode
+
+## Additional tools
+
+### containerd-stress
+In addition to `go test`-based testing executed via the `Makefile` targets, the `containerd-stress` tool is available and built with the `all` or `binaries` targets and installed during `make install`.
+
+With this tool you can stress a running containerd daemon for a specified period of time, selecting a concurrency level to generate stress against the daemon. The following command is an example of having five workers running for two hours against a default containerd gRPC socket address:
+
+```sh
+containerd-stress -c 5 -t 120
```

Comment:
anywhere to go for more details on containerd-stress?

Reply:
I wondered if it made sense to dump in `containerd-stress --help` here? At the moment that is all there is that I'm aware of (although the interface/options are really that simple). The original PR had a bit of detail: #1331

Reply:
If there's no other params than -c and -t, then it's probably good as is.   Or perhaps add in a "For more information, see `containerd-stress --help` somewhere.  That would guard against having to circle back and update this doc right away if an option is added later.  YMMV.

Reply:
done